### PR TITLE
docs: community extensions and examples

### PR DESCRIPTION
**Pull request by ruheni**
Rework the extensions-examples page to contain community packages and extension examples

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 1]
```diff
@@ -1,17 +1,39 @@
 ---
 title: 'Examples'
-metaTitle: 'Prisma Client Extensions | Examples'
-metaDescription: 'Extend the functionality of Prisma Client: Examples'
+metaTitle: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
+metaDescription: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
 ---
 
-<TopBlock>
+## Prisma-built extensions
```
Comment (Jolg42) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
## Extensions made by Prisma
```
Maybe? 🤔

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
+
+| Extension                                                                                             | Description                                                                                                               |
+| :---------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [`@prisma/extension-accelerate`](https://www.prisma.io/docs/data-platform/accelerate/getting-started) | Enables Accelerate, a global database cache                                                                               |
+| [`@prisma/extension-pulse`](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse)             | Enables Pulse, a service that captures change events from your database and delivers them instantly to your applications. |
+| [`@prisma/extension-read-replicas`](https://github.com/prisma/read-replicas)                          | Adds read replica support to Prisma Client                                                                                |
+
+## Community-built extensions
```
Comment (Jolg42) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
## Extensions made by Prisma's community
```
Maybe?

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 3]
```diff
@@ -1,17 +1,39 @@
...
 
-<TopBlock>
+## Prisma-built extensions
 
-The extensions are provided as examples only, and without warranty. We recommend using the examples as a source of inspiration for building your own extensions.
+The following is a list of extensions we've built at Prisma:
+
+| Extension                                                                                             | Description                                                                                                               |
+| :---------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [`@prisma/extension-accelerate`](https://www.prisma.io/docs/data-platform/accelerate/getting-started) | Enables Accelerate, a global database cache                                                                               |
```
Comment (Jolg42) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: I think we have a better description for accelerate?

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 4]
```diff
@@ -1,17 +1,39 @@
...
+| [`@prisma/extension-pulse`](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse)             | Enables Pulse, a service that captures change events from your database and delivers them instantly to your applications. |
+| [`@prisma/extension-read-replicas`](https://github.com/prisma/read-replicas)                          | Adds read replica support to Prisma Client                                                                                |
```
Comment (Jolg42) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
| [`@prisma/extension-read-replicas`](https://github.com/prisma/extension-read-replicas)                          | Adds read replica support to Prisma Client                                                                                |
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 5]
```diff
@@ -1,17 +1,39 @@
...
 
-<TopBlock>
+## Extensions made by Prisma
 
-The extensions are provided as examples only, and without warranty. We recommend using the examples as a source of inspiration for building your own extensions.
+The following is a list of extensions we've built at Prisma:
+
+| Extension                                                                                             | Description                                                                                                               |
+| :---------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [`@prisma/extension-accelerate`](https://www.prisma.io/docs/data-platform/accelerate/getting-started) | Enables Accelerate, a global database cache                                                                               |
```
Comment (ankur-arch) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
| [`@prisma/extension-accelerate`](https://www.prisma.io/docs/data-platform/accelerate/getting-started) | Enables Accelerate, a global database cache available in 300+ locations with built-in connection pooling                                                                           |
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx (hunk 3 above)
Comment (ankur-arch) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: @Jolg42 I've added a suggestion below that should suffice for a simple description.

**Review by Jolg42 — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 6]
```diff
@@ -1,17 +1,39 @@
...
+The following is a list of extensions we've built at Prisma:
+
+| Extension                                                                                             | Description                                                                                                               |
+| :---------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [`@prisma/extension-accelerate`](https://www.prisma.io/docs/data-platform/accelerate/getting-started) | Enables Accelerate, a global database cache available in 300+ locations with built-in connection pooling                                                                           |
+| [`@prisma/extension-pulse`](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse)             | Enables Pulse, a service that captures change events from your database and delivers them instantly to your applications. |
```
Comment (nikolasburk) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: are the repos for the extensions public? if so, could we link the `@prisma/extension-pulse` to the repo and and then link to the docs from the "product name"?

```suggestion
| [`@prisma/extension-pulse`](link-to-github)             | Enables [Pulse](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse), a service that captures change events from your database and delivers them instantly to your applications. |
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 7]
```diff
@@ -1,17 +1,39 @@
 ---
 title: 'Examples'
-metaTitle: 'Prisma Client Extensions | Examples'
-metaDescription: 'Extend the functionality of Prisma Client: Examples'
+metaTitle: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
+metaDescription: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
metaTitle: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
metaDescription: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx (hunk 6 above)
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: Even if not, "Accelerate" and "Pulse" should definitely link somewhere useful.

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 8]
```diff
@@ -1,17 +1,39 @@
...
+| [`prisma-extension-bark`](https://github.com/deptyped/prisma-extension-pagination)             | The extension is implementation of the Materialized Path pattern that allows you to easily create and interact with tree structures in Prisma. |
+| [`prisma-cursorstream`](https://github.com/etabits/prisma-cursorstream)                        | Adds cursor-based streaming                                                                                                                    |
+| [`prisma-gpt`](https://github.com/aliyeysides/prisma-gpt)                                      | Let's you query your database using natural language.                                                                                          |
+| [`prisma-extension-caching`](https://github.com/isaev-the-poetry/prisma-extension-caching)     | Transforms SQL data from queries in streams to improve performance in larger datasets                                                          |
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx:30]: Mention that we like PRs to this page to add your extension.

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 9]
```diff
@@ -1,17 +1,39 @@
...
 
 ## Examples
 
+<Admonition>
+
+The extensions are provided as examples only, and without warranty. We recommend using the examples as a source of inspiration for building your own extensions.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
These example extensions are provided as examples only, and without warranty. They are supposed to show how Prisma Client extensions can be created, using approaches documented here. We recommend using the examples as a source of inspiration for building your own extensions.
```

**Review by janpio — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx (hunk 6 above)
Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: The repositories are private. I'll link to the product pages instead.

Comment (jharrell) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: linking to npm is maybe an option as well?

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 10]
```diff
@@ -1,17 +1,41 @@
 ---
 title: 'Examples'
-metaTitle: 'Prisma Client Extensions | Examples'
-metaDescription: 'Extend the functionality of Prisma Client: Examples'
+metaTitle: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
+metaDescription: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
```
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
metaTitle: 'Prisma Client extensions | Prisma & Community built extensions and examples'
metaDescription: 'Prisma Client extensions | Prisma & Community built extensions and examples'
```
This looks better to my eyes but no clue if it's right. take it or leave it

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 11]
```diff
@@ -1,17 +1,41 @@
...
+| [`prisma-gpt`](https://github.com/aliyeysides/prisma-gpt)                                      | Let's you query your database using natural language.                                                                                          |
+| [`prisma-extension-caching`](https://github.com/isaev-the-poetry/prisma-extension-caching)     | Transforms SQL data from queries in streams to improve performance in larger datasets                                                          |
+
+If you have built an extension and would like to see it featured, feel free to add it to the list by opening a pull request.
 
 ## Examples
 
+<Admonition>
+
+These example extensions are provided as examples only, and without warranty. They are supposed to show how Prisma Client extensions can be created, using approaches documented here. We recommend using the examples as a source of inspiration for building your own extensions.
```
Comment (jharrell) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
The following example extensions are provided as examples only, and without warranty. They are supposed to show how Prisma Client extensions can be created using approaches documented here. We recommend using these examples as a source of inspiration for building your own extensions.
```
Nit picking. I'm happy with either option

**Review by jharrell — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx (hunk 10 above)
Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: In German, any two words that make one word that are separated by a space is a "Deppenleerzeichen". But that's German... so no idea what you are allowed to do in English :D

Comment (jharrell) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: You can do this in English, but the syntax of having the prepended "Prisma-" before the ampersand struck my eye as odd

Comment (nikolasburk) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: I would suggest to rephrase this entirely. 

Also: is it intentional that `metaTitle` and `metaDescription` are identical here? I would imagine this looks odd when shared on social and also misses an opportunity to share more context about this page via the `metaDescription`. 

I would suggest using different phrases for each and using the `metaDescription` to provide more context. 

```suggestion
metaTitle: 'Prisma Client extensions | Shared packages & examples'
metaDescription: 'Explore the Client extensions that have been built by Prisma and its community'
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 12]
```diff
@@ -1,17 +1,41 @@
...
+
+## Extensions made by Prisma's community
+
+The following is a list of extensions created by the community. If you want to create your own package, refer to the [Shared Prisma Client extensions](./shared-extensions) documentation.
+
+| Extension                                                                                      | Description                                                                                                                                    |
+| :--------------------------------------------------------------------------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`prisma-extension-supabase-rls`](https://github.com/dthyresson/prisma-extension-supabase-rls) | Adds support Supabase Row Level Security with Prisma                                                                                           |
```
Comment (petradonka) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]: I'd suggest making the descriptions more consistent. Also "Adds support Supabase Row Level Security with Prisma    " has a typo, missing a "for" I think, "add support for".

Comment (janpio) [reply content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
| [`prisma-extension-supabase-rls`](https://github.com/dthyresson/prisma-extension-supabase-rls) | Adds support for Supabase Row Level Security with Prisma                                                                                           |
```

File: content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx [hunk 13]
```diff
@@ -1,17 +1,41 @@
...
+| [`prisma-extension-bark`](https://github.com/deptyped/prisma-extension-pagination)             | The extension is implementation of the Materialized Path pattern that allows you to easily create and interact with tree structures in Prisma. |
```
Comment (nikolasburk) [inline content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx]:
```suggestion
| [`prisma-extension-bark`](https://github.com/deptyped/prisma-extension-pagination)             | Implements the Materialized Path pattern that allows you to easily create and interact with tree structures in Prisma |
```

**Pull request merged**